contracts/apps/ownership/OwnershipApp.sol

```diff
@@ -96,8 +124,8 @@ contract OwnershipApp is Application, Controller, Requestor {
   }
 
   function isHolder(address _holder) constant returns (bool) {
-    uint tokenCount = tokens.length;
-    for (uint i = 0; i < tokenCount; i++) {
+    uint tokenCount = getTokenCount();
```

Comment:
This is a small nitpick, but won't this be functionally equivalent to before, but will now incur the gas cost of the function call?